experiment_scheduler/master/master.py

```diff
@@ -293,6 +315,10 @@ def execute_task(self, task_manager):
                 "task": prior_task,
                 "task_manager": task_manager,
             }
+            task_obj = Task.get(id=prior_task_id)
+            task_obj.status = TaskStatus.Status.RUNNING
+            task_obj.task_manager_id = TaskManager.get(address=task_manager).id
+            task_obj.commit()
```

Comment:
Need Try Except in case of failure
```python
try:
   task_obj = Task.get(id=prior_task_id)
            task_obj.status = TaskStatus.Status.RUNNING
            task_obj.task_manager_id = TaskManager.get(address=task_manager).id
            task_obj.commit()
except {some exception} as e:
            task_obj.rollback()
```